Rentit-backend/config.env

```diff
@@ -3,4 +3,5 @@ PORT=2000
 DATABASE =mongodb+srv://a5h101:<PASSWORD>@cluster0.cyfx5wk.mongodb.net/Rentit?retryWrites=true&w=majority
```

Comment:
You have pushed `config.env` file here. You should not keep it publicly available. Delete it from here and add file in `gitignore`.